src/hexkit/protocols/objstorage.py

```diff
@@ -402,6 +413,14 @@ async def _get_object_download_url(
         """
         ...
 
+    @abstractmethod
+    async def _get_object_etag(self, *, bucket_id: str, object_id: str) -> str:
+        """Returns the etag of an object.
```

Comment:
```suggestion
        """Return the etag of an object.
```

Top line should use imperative mood - https://peps.python.org/pep-0257/#one-line-docstrings